api/segments/models.py

```diff
@@ -164,17 +164,19 @@ class Condition(AbstractBaseExportableModel):
     operator = models.CharField(choices=CONDITION_TYPES, max_length=500)
     property = models.CharField(blank=True, null=True, max_length=1000)
     value = models.CharField(max_length=1000, blank=True, null=True)
+    description = models.TextField(blank=True, null=True, max_length=2000)
 
     rule = models.ForeignKey(
         SegmentRule, on_delete=models.CASCADE, related_name="conditions"
     )
 
     def __str__(self):
-        return "Condition for %s: %s %s %s" % (
+        return "Condition for %s: %s %s %s %s" % (
             str(self.rule),
             self.property,
             self.operator,
             self.value,
+            self.description,
```

Comment:
I don't think we should add the description to the `__str__` response - the `__str__` method should return a succinct description of the entity. The description is more like metadata that we don't really care about here.